django_mailbox/models.py

```diff
@@ -326,6 +331,8 @@ def _get_dehydrated_message(self, msg, record):
 
     def _process_message(self, message):
         msg = Message()
+        if STORE_ORIGINAL_MESSAGE:
+            msg.eml.save('message.eml', ContentFile(message), save=False)
```

Comment:
Although Django will take care of the details for us, I'm pretty sure that this will cause the saved e-mail files to collide every time; you might want to instead `import uuid` at the top of the file and instead set the filename like:

```python
msg.eml.save('%s.eml' % uuid.uuid4(), ContentFile(message), save=False)
```